sw/airborne/subsystems/ins/ins_int.c

```diff
@@ -145,6 +141,15 @@ static void gps_cb(uint8_t sender_id, uint32_t stamp, struct GpsState *gps_s);
 static abi_event vel_est_ev;
 static void vel_est_cb(uint8_t sender_id, uint32_t stamp, float x, float y, float z, float noise);
 
+/** ABI binding for AGL.
+ * Usually this is coming from opticflow.
```

Comment:
This sounds a bit weird, optiflow doesn't give you an AGL measurement... it's still the sonar (or something similar)